src/QuickMarcEditor/QuickMarcEditorRows/utils.js

```diff
@@ -16,7 +17,15 @@ const READ_ONLY_ROWS = new Set(['001', '005']);
 
 const READ_ONLY_ROWS_FOR_DUPLICATE = new Set([LEADER_TAG, '001', '005']);
 
-export const isReadOnly = (recordRow, action = QUICK_MARC_ACTIONS.EDIT) => {
+export const isReadOnly = (
+  recordRow,
+  action = QUICK_MARC_ACTIONS.EDIT,
+  marcType = MARC_TYPES.BIB,
+) => {
+  if (marcType === MARC_TYPES.HOLDINGS) {
```

Comment:
I think it's better to create a separate constant and call it `READ_ONLY_ROWS_FOR_HOLDINGS` and not mutate an existing object because it could cause bugs that would be difficult to debug.
For example, if we edit a holdings record first and the go to a bib record and try to edit it - would 004 still be editable or not?

Reply:
Good point, thanks! Fixed it.